portal/home.php

```diff
@@ -190,7 +219,65 @@ function saveProfile(){
                 <div class="col-sm-6">
                     <div class="panel panel-primary collapse" id="appointmentpanel">
                         <header class="panel-heading"><?php echo xlt('Appointments'); ?>  </header>
-                        <div id="appointmentslist" class="panel-body"></div>
+                        <div id="appointmentslist" class="panel-body">
+                        <?php
+                            $query = "SELECT e.pc_eid, e.pc_aid, e.pc_title, e.pc_eventDate, " . "e.pc_startTime, e.pc_hometext, e.pc_apptstatus, u.fname, u.lname, u.mname, " .
```

Comment:
Does this capture recurrent appt. Guessing it doesn't. @epsdky added really nice support for this in the patient summary screen. Rec checking that code out to use those functions that will grab all appt you want (in addition to the recurrent ones).

Reply:
I can do that.